README.md

```diff
@@ -4,36 +4,10 @@ The goal of this project is to aggregate multiple sources of addresses and then
 
 The big part of this project being the deduplication process and cleaning the data.
 
-## Importers
+## Workflow
 
-Each importer generates an [sqlite](https://www.sqlite.org/index.html) `addresses.db` file with the following tables:
+It first imports the addresses from the importers. You might want to take a look at them directly if you want more information on a specific importer.
```

Comment:
There was a pleaunasm :D

```suggestion
It first loads addresses data using the importers. You might want to take a look in `[importers](/importers)/{source name}` if you want more information on a specific importer.
```

Maybe it could be fancy (and useful?) to list all importers explicitly here with a link to the corresponding README and source?

Reply:
Not sure if it's a good idea. All the importers are visible in the importers folder after all...